src/components/CountrySelector.js

```diff
@@ -0,0 +1,77 @@
+import React, {useEffect} from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import styles from '../styles/styles';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+import useLocalize from '../hooks/useLocalize';
+import MenuItemWithTopDescription from './MenuItemWithTopDescription';
+import FormHelpMessage from './FormHelpMessage';
+
+const propTypes = {
+    /** Form error text. e.g when no country is selected */
+    errorText: PropTypes.string,
+
+    /** Callback called when the country changes. */
+    onInputChange: PropTypes.func.isRequired,
+
+    /** Current selected country  */
+    value: PropTypes.string,
+
+    /** inputID used by the Form component */
+    // eslint-disable-next-line react/no-unused-prop-types
+    inputID: PropTypes.string.isRequired,
+
+    /** React ref being forwarded to the MenuItemWithTopDescription */
+    forwardedRef: PropTypes.func,
+};
+
+const defaultProps = {
+    errorText: '',
+    value: '',
```

Comment:
Setting an empty string as default prevents the `Form` from passing the `value`. it has to be `undefined`
```
value: undefined
```

Reply:
Updated!